// global constants
// can't be declared ass constants du to initalation prosses


extern std::string SR_BASE_URI;
extern std::string SR_BASE_URI_HTTPS;
extern std::string PROVIDER_ADDRESS;
extern std::string PROVIDER_ADDRESS6;
extern std::string CUSTOM_URL;
extern std::string PROVIDER_SYSTEM_NAME;
extern std::string SERVICE_DEFINITION;
extern std::string INTERFACE;
extern std::string PRIVATE_KEY_PATH;
extern std::string PUBLIC_KEY_PATH;

extern int PROVIDER_PORT;

extern bool SECURE_ARROWHEAD_INTERFACE;
extern bool SECURE_PROVIDER_INTERFACE;
